src/main/java/datacollector/Demo.java

```diff
@@ -0,0 +1,106 @@
+package datacollector;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentSnippet;
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.api.services.youtube.model.ResourceId;
+import com.google.api.services.youtube.model.SearchListResponse;
+import com.google.api.services.youtube.model.SearchResult;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.common.collect.Lists;
+
+public class Demo{
+	private static YouTube youtube;
+	//private static YouTube youtubeComments;
+	private static String apiKey = "AIzaSyCerMe9K8HTxXV_-LO52Vbe7miAWyC1-08";
```

Comment:
api key ... should not be committed in the source code.

Use environment variable or some other way to mask your api key away.